### PR TITLE
Use importmaps

### PR DESCRIPTION
**Pull request by noplisu**
## Pull Request Summary

Before we update the page to rails 7 we want to bring the page to the newest standards. Here we move away from webpacker and yarn in favor of using importmaps.

## Description of the Changes

- We switched the webpack gem to importmaps
- We remove yarn and package.json
- We convert coffescript files to javascript
- Fix scripts not loading correctly on page
- Remove the use of jQuery in map generation script

### CODE REVIEW

File: bin/spring
Comment (torrocus) [inline bin/spring:1]: Why do we disable execution attribute for spring script?

Comment (torrocus) [reply bin/spring:1]: IMO Spring is rather related to Rails, so we don't need to change anything here.

File: app/javascript/fractalsoft/cieszyn.js [hunk 1]
```diff
@@ -14,35 +16,29 @@ const gliwice = {
...
+    marker.bindPopup(location.title).openPopup();
   }
 }
 
-const locationMap = () => new LocationMap(cieszyn);
-
-jQuery(() => locationMap());
+document.addEventListener("turbolinks:load", function() {
+  const locationMap = () => new LocationMap(cieszyn);
+  console.log("Reload map");
```
Comment (torrocus) [inline app/javascript/fractalsoft/cieszyn.js:42]: Looks like something left after debugging process. :smile:

**Review by torrocus — APPROVED** [review]
Overall it looks good. A few questions and minor comments, but nothing blocking.

File: app/javascript/fractalsoft/contact_forms.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-jQuery(function() {
+document.addEventListener("turbolinks:load", function() {
   $('.btn-map').click(function() {
```
[the remark this answers is not in the document]
Comment (noplisu) [reply app/javascript/fractalsoft/contact_forms.js:2]: It is our class. I would probably suggest even changing it to show it is a class used in a script to something like `js-btn-map`. But I tried to do only minimal changes to the scripts at the moment.

**Review by torrocus — APPROVED** [review]

**Pull request merged**